src/components/partials/help-guide-lb-ingredient.js

```diff
@@ -0,0 +1,1151 @@
+import { html } from 'lit-html';
+import { modifyAccordionState } from '../../scripts/collapse-expand';
+/* eslint-disable no-irregular-whitespace */
+
+export default () => html`
+<div class="side-padding vertical-padding">
+    <h4>General requirements</h4>
+    <section class="qg-accordion flex-column" aria-label="Accordion Label">
+        <div class="controls-group">
+            <input type="radio" name="control" id="collapse" class="controls collapse" value="collapse" role="radio">
+
+            <label for="collapse" class="controls" @click=${e => modifyAccordionState(e, false)}>Collapse all</label>
+
+            <span class="controls">|</span>
+
+            <input type="radio" name="control" id="expand" class="controls expand" value="expand" role="radio">
+
+            <label for="collapse" class="controls" @click=${e => modifyAccordionState(e, true)}>Expand all</label>
+        </div>
+
+        <article>
+            <input id="ingredient-list" type="checkbox" name="tabs" tabindex="-1" aria-controls="id-panel-content-1"
+                aria-expanded="false" role="checkbox" />
+            <h3 class="acc-heading">
+                <label for="ingredient-list">
+                    <span class="title">
+                        What is an ingredient list?
+                    </span>
+                    <span class="arrow"><i></i></span>
+                </label>
+            </h3>
+
+            <div class="collapsing-section" aria-hidden="true" id="id-panel-content-1">
+                <p>
+                    The ingredient list is a statement of ingredients that are used to make the food. The ingredient
```

Comment:
why is large amounts of html inside a js file, should this content be pulled from squiz as static assets or from form.io questionnaire?

Reply:
That isn't how modern front end development is done.